R/autoplot.permustats.R

```diff
@@ -0,0 +1,97 @@
+### fortify permustats objects
+
+#' Autoplot Graphics for vegan permustats Objects
+#'
+#' Alternatives for \pkg{lattice} graphics functions
+#' \code{\link[vegan]{densityplot.permustats}},
+#' \code{\link[vegan]{qqmath.permustats}} and
+#' \code{\link[vegan]{boxplot.permustats}}.
+#'
+#' Function \code{fortify} returns a data frame with variables
+#' \code{Permutations} (numeric) and \code{Term} (factor labelling the
+#' permutation). The result of \code{fortify} can be used to custom
+#' build diagnostic plots. \code{autoplot} provides quick basic graphs
+#' with limited flexibility.
+#'
+#' @param object Result object from \code{\link[vegan]{permustats}}.
+#' @param plot Plot type, or geometry of \pkg{ggplot2}.
+#' @param scale Use standardized effect sizes (SES).
+#' @param facet Split graph to facets by \code{Term}.
+#' @param gg.params Arguments passed to function drawing the box-like
+#'     object. Depending on argument \code{plot} the parameters are
+#'     passed to
+#'     \code{\link[ggplot2]{geom_boxplot}},\code{\link[ggplot2]{geom_violin}},
+#'     \code{\link[ggplot2]{geom_density}} or
+#'     \code{\link[ggplot2]{geom_qq}}
+#' @param ... Other parameters passed to functions (ignored).
+#'
+#' @examples
+#' data(dune, dune.env)
+#' mod <- cca(dune ~ A1 + Management + Moisture, dune.env)
+#' (ano <- anova(mod, by="onedf"))
+#' pstat <- permustats(ano)
+#' head(fortify(pstat))
+#' autoplot(pstat, "box")
+#' ## avoid overplotting x-axis text
+#' autoplot(pstat, "violin") + scale_x_discrete(guide = guide_axis(n.dodge = 2))
+#' autoplot(pstat, "density", facet = TRUE)
+#' autoplot(pstat, "qqnorm", facet = TRUE) + geom_qq_line()
+#'
+
+#' @importFrom ggplot2 fortify ggplot aes_ geom_hline geom_vline geom_boxplot
```

Comment:
Will need to import `.data` from package *dplyr* (see below) and `tibble::tibble` (see below too) and can delete `aes_`

```r
#' @importFrom dplyr .data
#' @importFrom tibble tibble
```

I think that means we'll need to modify `DESCRIPTION` also to add *tibble* as an imports pkg.